docs/ops/logical/LogicalAnd_1.md

```diff
@@ -6,6 +6,14 @@
 
 **Short description**: *LogicalAnd* performs element-wise logical AND operation with two given tensors applying multi-directional broadcast rules.
 
+**Detailed description**: Before performing logical operation, input tensors *a* and *b* are broadcasted if their shapes are different and `auto_broadcast` attributes is not `none`. Broadcasting is performed according to `auto_broadcast` value.
+
+After broadcasting *LogicalAnd* does the following with the input tensors *a* and *b*:
+
+\f[
+o_{i} = a_{i} and b_{i}
+\f]
+
 **Attributes**:
 
 * *auto_broadcast*
```

Comment:
`auto_broadcast ` attribute shall be aligned with description as in Add [operator](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/arithmetic/Add_1.md)

Reply:
updated